samples/test-runner/build.gradle.kts

```diff
@@ -29,12 +32,16 @@ android {
 
 dependencies {
     androidTestImplementation("com.avito.android:test-inhouse-runner")
+    androidTestImplementation("com.avito.android:test-report")
+    androidTestImplementation("com.avito.android:logger")
     androidTestImplementation(libs.truth)
 
     androidTestUtil(libs.testOrchestrator)
 }
 
 instrumentation {
+    output = "$buildDir/reports/test-runner"
```

Comment:
using `buildDir` is much much better choice for local development

Reply:
We can use it for CI build also

Reply:
yep